Makefile

```diff
@@ -30,7 +30,11 @@ openapi:
 	./etc/generate-openapi.sh
 
 fmt: $(SOURCES_NO_VENDOR)
+	# Format everything, but the import-format doesn't match our desired pattern.
 	gofmt -w -s $^
+	# Remove unused imports.
+	go run golang.org/x/tools/cmd/goimports -w $^
```

Comment:
This is a kludge to work around a bad piece of codegen.

The default codegen for `format: binary` pulls the full response body into memory and massages it into a `*os.File`. I was able to change the template to skip all that and return a direct `io.ReadCloser`, but I wasn't able to prevent the now-unneeded `import "os"` from appearing without breaking imports in other generated files.

`goimports` seems to be the only tool that will detect & remove unused imports. Luckily it seems to run quickly.